packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -61,6 +62,19 @@ abstract class MaterialLocalizations {
   /// Label for the [AboutBox] button that shows the [LicensePage].
   String get viewLicensesButtonLabel;
 
+  /// Provides geometric text preferences for the current locale.
+  ///
+  /// This text theme is incomplete. For example, it lacks text color
+  /// information. This theme must be merged with another text theme that
+  /// provides the missing values. The text styles provided by this theme are
```

Comment:
Does "The text styles provided by this theme are inherited" mean that localTextGeometry's TextStyles will be inherit:true, or inherit:false? I think we should be more explicit about that.

Reply:
Done.